Formula/php.rb

```diff
@@ -0,0 +1,333 @@
+class Php < Formula
+  desc "General-purpose scripting language"
+  homepage "https://php.net/"
+
+  stable do
+    version "7.1.10"
+    url "http://php.net/get/php-7.1.10.tar.gz/from/this/mirror"
+    sha256 "edc6a7c3fe89419525ce51969c5f48610e53613235bbef255c3a4db33b458083"
+
+    depends_on "libtool" => :run
+    depends_on "mcrypt"
+  end
+
+  devel do
+    url "https://downloads.php.net/~remi/php-7.2.0RC3.tar.gz"
+    sha256 "076491818dd80d723c9232bd4553a95eb30374f3c602d5ca67b95f2ac5e1655a"
+
+    depends_on "argon2"
+    depends_on "libsodium"
+  end
+
+  option "with-thread-safety", "Build with thread safety"
+
+  depends_on "aspell"
+  depends_on "curl" if MacOS.version < :lion
+  depends_on "enchant"
+  depends_on "freetds"
+  depends_on "freetype"
+  depends_on "gettext"
+  depends_on "gmp"
+  depends_on "httpd"
+  depends_on "icu4c"
+  depends_on "imap-uw"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libpq"
+  depends_on "libzip"
+  depends_on "net-snmp"
+  depends_on "openssl"
+  depends_on "pcre"
+  depends_on "tidy-html5"
+  depends_on "unixodbc"
+  depends_on "webp"
+
+  needs :cxx11
+
+  def install
+    ENV.cxx11
+
+    # Fix missing header file during configure for libzip include
```

Comment:
This needs to be fixed upstream.

Reply:
libzip includes a pkgconfig file which lists both include directories. I'll see if I can get brew to work with that instead - would that be acceptable ?

Reply:
I can get pkg-config to detect libzip but it still doesn't seem to pick up on the additional include directory from this
```
prefix=/usr/local/Cellar/libzip/1.3.0
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include
libincludedir=${exec_prefix}/lib/libzip/include

zipcmp=/usr/local/Cellar/libzip/1.3.0/bin/zipcmp

Name: libzip
Description: library for handling zip archives
Version: 1.3.0
Libs:  -L${libdir} -lzip -lbz2 -lz
Cflags: -I${includedir} -I${libincludedir}
```

Reply:
Works for me by adding the pkg-config build dependency _and_ building the zip extension as shared.